sirepo/job_supervisor.py

```diff
@@ -153,8 +153,7 @@ def init():
             sequential=(.1, float, 'maximum run-time for sequential job'),
         ),
         purge_free_after_days=(1000, int, 'how many days to wait before purging a free users simulation'),
-        purge_free_start=('02:00:00', str, 'time to first start purging free users simulations (%H:%M:%S)'),
-        purge_free_period=('01:00:00', str, 'how often to purge free users simulations after start time (%H:%M:%S)'),
+        purge_free_period=('12:00:00', str, 'how often to purge free users simulations (%H:%M:%S)'),
```

Comment:
I don't like the name purge_free_period.  `purge_non_premium_task_secs` and `purge_non_premium_after_days`. 

It's ridiculous there isn't a better way to parse.

BTW, there should be a _cfg_function that does this conversion. The default value should be None, btw, and we shouldn't purge simulations if it is None.

```py
purge_non_premium_task_secs=(None, _cfg_secs, 'when to clean up simulation runs of free users (%H:%M:%S)'),
```

with this here for now, but could maybe get moved into pkconfig:

```py
def _cfg_secs(value):
    try:
        return int(secs)
    except Exception:
        pass
    t = datetime.datetime.strptime(value, '%H:%M:%S') # single quotes, not double (as copied :)
    return int(...total_seconds())
```

note the `int` on `total_seconds` which returns a `float`.

Reply:
https://github.com/radiasoft/sirepo/pull/2485/commits/8cb87e0aaa7ead347b40f12da4f6895376ca1e92